### ISSUE TITLE
tabindex incorrectly applied to a11y-hideable elements

### ISSUE THREAD

**Issue by tomgreenfield** [adaptlearning/adapt_framework#1093]
From https://github.com/adaptlearning/adapt_framework/pull/1092#discussion_r66070023:

> Some elements, which were never explicitly in the tab order, need to be removed from the tab order when a popup occurs. Label tags associated with input tags are an example of such elements. When the popdown occurs their tabindex="0" should be removed as it wouldn't have originally existed and will cause a double tab if left.

>iFrame containers for youtube videos will also have this same problem.
